docs/examples/kube2iam.rst

```diff
@@ -0,0 +1,247 @@
+        value = "${aws_iam_policy.kube2iam.arn}"
+    }
+
+    output "kube2iam_path" {
+        value = "/kube2iam_${var.cluster_name}/"
+    }
+
+
+You can run the Terraform project the following way:
+
+.. code-block:: bash
+
+    terraform init
+    terraform apply -var cluster_name=example -var region=eu-west-1
+
+Attach instance policy
+++++++++++++++++++++++
+
+Add the created IAM policy ARN to your tarmak config. You can do this by
+adding `additional IAM policies <https://docs.tarmak.io/user-guide.html#additional-iam-policies>`_.
+
+Deploy kube2iam
++++++++++++++++
+
+With `HELM <https://www.helm.sh/>`_ it is easy to deploy kube2iam with 
+the correct settings.
+
+You can deploy it with the following command:
+
+.. code-block:: bash
+
+    helm upgrade kube2iam stable/kube2iam \
+    --install \
+    --version 0.9.0 \
+    --namespace kube-system \
+    --set=extraArgs.host-ip=127.0.0.1 \
+    --set=extraArgs.log-format=json \
+    --set=updateStrategy=RollingUpdate \
+    --set=rbac.create=true \
+    --set=host.iptables=false
+
+
+We set ``iptables`` to false and ``host-ip`` to 127.0.0.1 as Tarmak already creates
+the iptables rule and forward it to ``127.0.0.1:8181``.
+Specific kube2iam options can be found in the `documentation <https://github.com/jtblin/kube2iam>`_ of kube2iam.
+
+Usage
+~~~~~
+
+Now that kube2IAM is installed on your system, you can start creating roles
+and policies to give your pods access to AWS resources.
+
+An example creation of an IAM policy and role:
+
+.. code-block:: none
+
+    terraform {}
+
+    provider "aws" {
+        region = "${var.aws_region}"
+    }
+
+    variable "aws_region" {
+        description = "AWS Region you want to deploy it in"
+        default     = "eu-west-1"
+    }
+
+    variable "cluster_name" {
+        description = "Name of the cluster"
+    }
+
+    variable "instance_iam_role_arn" {
+        description = "ARN of the instance IAM role"
+    }
+
+    resource "aws_iam_role" "test_role" {
+        name = "test_role"
+        path = "/kube2iam_${var.cluster_name}/"
+
+        assume_role_policy = "${data.aws_iam_policy_document.test_role.json}"
+    }
+
+    data "aws_iam_policy_document" "test_role" {
+        statement {
+            sid = "1"
+
+            actions = [
+                "sts:AssumeRole",
+            ]
+
+            principals {
+                type        = "AWS"
+                identifiers = ["${var.instance_iam_role_arn}"]
+            }
+        }
+    }
+
+    resource "aws_iam_role_policy" "test_role_policy" {
+        name = "test_policy"
+        role = "${aws_iam_role.test_role.id}"
+
+        policy = "${data.aws_iam_policy_document.test_role_policy.json}"
+    }
+
+    data "aws_iam_policy_document" "test_role_policy" {
+        statement {
+            sid = "1"
+
+            actions = [
+                "s3:ListAllMyBuckets",
+            ]
+
+            resources = [
+                "*",
+            ]
+        }
+    }
```

Comment:
why are these in italics?

Reply:
No idea, when I generate the docs it looks ok to me.
<img width="461" alt="screen shot 2018-06-20 at 16 07 28" src="https://user-images.githubusercontent.com/2216724/41667148-2e1dec1a-74a4-11e8-8cbc-6bf26ebd95d3.png">